cypress/integration/homework16-female-gender.spec.js

```diff
@@ -0,0 +1,138 @@
+import 'cypress-file-upload';
+import { RegistrationFormPage } from '../page-objects/student-registration-form';
+import { SubmitStudentDataForm } from '../page-objects/submit-student-data-modal';
+
+import {
+  randomUserData,
+  GENDER,
+  SUBJECT,
+  HOBBIES,
+  STATES,
+  UTTAR_PRADESH_STATE_CITIES,
+} from '../constants/user-data';
+
+describe('Lesson16', () => {
+  describe('Female Gender Flow', () => {
```

Comment:
u can use context for the second describe

Reply:
good idea